packages/web-components/src/components/masthead/masthead.scss

```diff
@@ -499,3 +499,15 @@
 :host(#{$dds-prefix}-masthead-search-container) {
   display: contents;
 }
+
+:host(#{$dds-prefix}-megamenu-top-nav-menu)[active] a::after {
```

Comment:
If there is no issue with `data-selected` attribute, I think we can remove this as the `aria-selected` issue has already been addressed in a previous PR.